### PR TITLE
DATA: Parsifal-M added event FOSDEM 2023

### PR DESCRIPTION
**Pull request by Parsifal-M**
Signed-off-by: Peter Macdonald <macdonald.peter90@gmail.com>

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thanks, everyone! :+1: 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4413"></a>

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by kumarsonsoff3 — DISMISSED** [review]
Looks good, Thanks for your Contribution! 🎉

**Review by iamthecloverly — APPROVED** [review]

**Review by stevenovak123 — DISMISSED** [review]

File: data/Parsifal-M/events/2023-02-04-FOSDEM.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "color": "purple",
+    "name": "FOSDEM 2023",
+    "description": "Free and Open source Software Developers' European Meeting is a non-commercial, volunteer-organized European event centered on free and open-source software development. It is aimed at developers and anyone interested in the free and open-source software movement.",
+    "date": {
+      "start": "2022-02-04T16:00:00.000+00:00",
+      "end": "2022-02-05T17:00:00.000+00:00"
+    },
+    "url": "https://fosdem.org/2023/"
+  }
+
```
Comment (Panquesito7) [inline data/Parsifal-M/events/2023-02-04-FOSDEM.json:14]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

Comment (Parsifal-M) [reply data/Parsifal-M/events/2023-02-04-FOSDEM.json:14]: Good catch, will remediate 👍

File: data/Parsifal-M/events/2023-02-04-FOSDEM.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "color": "purple",
+    "name": "FOSDEM 2023",
+    "description": "Free and Open source Software Developers' European Meeting is a non-commercial, volunteer-organized European event centered on free and open-source software development. It is aimed at developers and anyone interested in the free and open-source software movement.",
+    "date": {
+      "start": "2022-02-04T16:00:00.000+00:00",
+      "end": "2022-02-05T17:00:00.000+00:00"
+    },
+    "url": "https://fosdem.org/2023/"
+}
+
```
Comment (Panquesito7) [inline data/Parsifal-M/events/2023-02-04-FOSDEM.json:14]: I think it's still missing. 😅
```suggestion
}

```

Comment (Parsifal-M) [reply data/Parsifal-M/events/2023-02-04-FOSDEM.json:14]: 🤔 Not sure I follow, there is an extra new line there after the closing bracket. What am I missing 😢

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by stevenovak123 — APPROVED** [review]

**Pull request merged**